constants/creator_groups.json

```diff
@@ -347,7 +368,15 @@
     "WeChat AI",
     "Wind Information Co.",
     "Yahoo! Inc.",
-    "Zhihu Inc"
+    "Zhihu Inc",
+    "AiCure",
+    "Meitu Inc.",
+    "Toyota",
+    "Twenty Billion Neurons GmbH",
+    "Naver",
```

Comment:
NAVER AI Lab is already under Industry research labs. Alibaba is as well. Please make sure we are not duplicating. It allows us to make sure we can attribute all creations to the same creator, without having 6 different names for them.